libraroo-backend/src/auth/auth.service.ts

```diff
@@ -1,14 +1,15 @@
 import { User } from './../users/models/user.model';
 import { UsersRepository } from '../users/users.repository';
 import { LoginDto } from '../auth/dto/login.dto';
-import { BadRequestException, Injectable } from '@nestjs/common';
+import { BadRequestException, Injectable, Logger } from '@nestjs/common';
 import { LoginResponseDto } from './dto/login-response.dto';
 import { compare } from 'bcrypt';
 import { JwtService } from '@nestjs/jwt';
 import { JwtPayLoad } from '../shared/jwt/jwt-payload';
 
 @Injectable()
 export class AuthService {
+  logger: Logger = new Logger(AuthService.name);
```

Comment:
Shouldn't this be private if its called only from the service itself? same with the rest of loggers